scripts/actions/internal_users

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC STARTING
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source scripts/_common.sh
+source /usr/share/yunohost/helpers
+
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
+
+# Get authentication backend and set as lowercase
+#auth_backend=${YNH_ACTION_IS_INTERNAL_USERS,,}
+# Get "is_internal_users" and set as lowercase
+is_internal_users=${YNH_ACTION_IS_INTERNAL_USERS,,}
+
+# Get the full name of the app, Example: strut__3
+app=$YNH_APP_INSTANCE_NAME
+
+admin=$(ynh_app_setting_get $app admin)
+is_public=$(ynh_app_setting_get $app is_public)
+final_path=$(ynh_app_setting_get $app final_path)
+
+auth_backend_old=$(ynh_app_setting_get $app auth_backend)
+is_internal_users_old=$(ynh_app_setting_get $app is_internal_users)
+
+#=================================================
+# CHECK IF ARGUMENTS AND REQUIREMENTS ARE CORRECT
+#=================================================
+
+#=================================================
+# CHECK IF AN ACTION HAS TO BE DONE
+#=================================================
+
+if [ $is_internal_users -eq $is_internal_users_old ]
+then
+    ynh_die "is_internal_users is already set as $is_internal_users." 0
+fi
+
+# Ensure that app is public
+if [ $is_public -eq 0 ]; then
+    ynh_die "Wiki must be public if you want your people to be able to reach it. Run 'public_private' and come back here" 1
+fi
+
+# Plugin "authchained" needs to be installed for this script to run
+if [ ! -d $final_path/lib/plugins/authchained ]; then
+    ynh_die "Plugin 'authchained' is not installed and must be installed beforehand." 1
+fi
```

Comment:
Small "typo", but those 2 previous check should be in "# CHECK IF ARGUMENTS AND REQUIREMENTS ARE CORRECT"